Packs/DigitalShadows/Integrations/ReliaQuestGreyMatterDRPIncidents/ReliaQuestGreyMatterDRPIncidents.yml

```diff
@@ -0,0 +1,143 @@
+  required: false
+  type: 8
+- additionalinfo: Remove all if you don't want to select  all risk types, and then select specifically
+  defaultvalue: all
+  display: Risk Types
+  name: riskTypes
+  options:
+  - association-with-malware
+  - custom-risk
+  - evidence-of-adversary-impact
+  - evidence-of-credential-access
+  - evidence-of-initial-access
+  - evidence-of-reconnaissance
+  - evidence-of-resource-development
+  - expired-certificate
+  - exposed-access-key
+  - exposed-cloud-service
+  - exposed-commercial-document
+  - exposed-credential
+  - exposed-customer-account
+  - exposed-device
+  - exposed-employee-pii
+  - exposed-payment-card
+  - exposed-technical-document
+  - impersonating-company-profile
+  - impersonating-domain
+  - impersonating-employee-profile
+  - impersonating-mobile-app
+  - impersonating-subdomain
+  - leaked-sensitive-code
+  - leaked-sensitive-technology
+  - marked-document
+  - mention-by-a-threat-actor
+  - misconfigured-web-server
+  - exposed-port
+  - phishing-report
+  - phishing-webpage
+  - report-of-botnet-activity
+  - report-of-technical-vulnerability
+  - reported-data-breach
+  - revoked-certificate
+  - selling-unauthorized-goods
+  - unauthorized-code-commit
+  - vulnerable-product
+  - vulnerable-service
+  - weak-certificate
+  - all
+  required: true
+  type: 16
+- additionalinfo: Remove all if you don't want to select  all risk types, and then select specifically
+  defaultvalue: all
+  display: Risk Level
+  name: riskLevel
+  options:
+  - all
+  - none
+  - low
+  - medium
+  - high
+  - very-low
+  - very-high
+  required: false
+  type: 16
+- additionalinfo: If you don't want to ingest rejected/resolved/closed incidents then set it to False. Otherwise incidents will ingested with auto-closed=True
+  defaultvalue: "true"
+  display: Ingest Rejected/Resolved/Closed Incidents
+  name: ingestClosedIncidents
+  required: false
+  type: 8
+- additionalinfo: No of incidents want to fetch per iteration
+  defaultvalue: "50"
+  display: Fetch Limit
+  name: fetchLimit
+  required: true
+  type: 0
+- additionalinfo: This controls how often the integration will perform a fetch_incidents command
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Since when want to fetch the data with given format(%Y-%m-%dT%H:%M:%SZ)
+  defaultvalue: "2020-01-20T00:00:00Z"
+  display: Start date
+  name: sinceDate
+  required: true
+  type: 0
+display: ReliaQuest GreyMatter DRP Incidents
+name: ReliaQuest GreyMatter DRP Incidents
+script:
+  commands:
+  - arguments:
+    - name: query
+      required: true
+    description: Perform a general search against incidents, threats closed sources, etc.
+    name: ds-search
+  dockerimage: demisto/python3:3.10.13.74666
+  isfetch: true
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
+fromversion: 5.0.0
+supportlevelheader: xsoar
+defaultmapperin: Reliaquest GreyMatter DRP Mapper
```

Comment:
Note that the mapper and classifier are mentioned incorrectly here
![image](https://github.com/demisto/content/assets/88268646/3673eded-7c90-43a0-9b07-a0d8c56df52e)

Reply:
yes will correct it